src/ufonormalizer.py

```diff
@@ -1090,6 +1090,8 @@ def _convertPlistElementToObject(element):
             else:
                 obj[key] = _convertPlistElementToObject(subElement)
     elif tag == "string":
+        if not element.text:
```

Comment:
actually, wait... shouldn't you check

```python
if element.txt == "":
    return ""
```

instead of `if not element.text`?

The latter, will be True also when `element.text == None`. 

I just checked what lmxl does (to have a reference) and it seems to distinguish between an element that has text `None` or one that has an empty text string:

```python
In [1]: from lxml import etree

In [2]: e = etree.Element("a")

In [3]: e.text

In [4]: etree.tostring(e)
Out[4]: b'<a/>'

In [5]: e.text = ""

In [6]: etree.tostring(e)
Out[6]: b'<a></a>'
```

Reply:
In this case, `element.text == None`.

Reply:
```
if element.txt == "":
    return ""
```
Wouldn't change anything because 
```
In [12]: ufonormalizer.xmlEscapeText("")
Out[12]: ''
```

Reply:
ok, it appeas that neither ElementTree nor lxml.etree distinguish between these two cases when parsing, they are always parsed as `Element.text == None`. Compare:

```python
>>> import xml.etree.ElementTree as ET
>>> assert ET.fromstring('<a></a>').text is None
>>> assert ET.fromstring('<a/>').text is None
>>> from lxml import etree
>>> assert etree.fromstring('<a/>').text is None
>>> assert etree.fromstring('<a></a>').text is None
```

In the case of ufonormalizer's XMLWriter, having a value being None however, means completely dropping the element, which produces an invalid plist, as Nikolaus noted.

So I think we are fine if we treat an element's text being None as an empty string, so we can write it back, like this PR is doing.